src/controller/user/Controller.ts

```diff
@@ -87,19 +104,30 @@ class User {
       const Id = req.params.id;
       const toDeleteUser = userData.find((item) => item.id === Id);
       userData.splice(userData.indexOf(toDeleteUser), 1);
-      return res
-        .status(200)
-        .send({
-          message: 'user deleted successfully',
-          data: userData,
-          deleted_User: toDeleteUser,
-        });
+      return res.status(200).send({
+        message: 'user deleted successfully',
+        data: userData,
+        deleted_User: toDeleteUser,
+      });
+    } catch (error) {
+      return res.status(500).send({ err: 'server error', message: 'Something went Wrong' });
+    }
+  }
+
+  // Create JWT token -----
+  createToken(req: Request, res: Response, next: NextFunction) {
+    try {
+      const { id, email } = req.body;
+      const token = jwt.sign({ id, email }, config.secret, { expiresIn: '10h' });
+      return res.status(200).send({
+        message: 'token successfully created',
+        data: { token },
+        status: 200,
+      });
     } catch (error) {
-      return res
-        .status(500)
-        .send({ err: 'server error', message: 'Something went Wrong' });
+      return res.status(500).send({ err: 'Server Error', message: 'Something went wrong' });
     }
   }
 }
 
```

Comment:
On try-catch is enough. If data is missing or condition fails then throw error with proper message. Use catch block to catch the error and return. No need return error using multiple time using res.send()